packages/gitmoji-changelog-cli/src/cli.e2e.spec.js

```diff
@@ -32,33 +31,115 @@ describe('generate changelog', () => {
     )
   })
 
-  afterAll(() => {
+  afterEach(() => {
     removeSync(testDir)
   })
 
-  it('should generate changelog even if user doesn\'t call cli at each tag', async () => {
-    await newVersion('1.0.0')
-    await gitmojiChangelog()
+  describe('init', () => {
+    it("should get a 1.0.0 version while initializing changelog by calling cli without arguments and having package.json's version set to 1.0.0", async () => {
+      await makeChanges('file1')
+      await commit(':sparkles: Add some file')
+      await bumpVersion('1.0.0')
+      gitmojiChangelog()
+      await commit(':bookmark: Version 1.0.0')
 
-    expect(getChangelog()).toMatch(/1.0.0/)
+      expect(getChangelog()).includes(['1.0.0'])
```

Comment:
You should keep a regexp or give the full expected line, ex: `## 2.0.0-alpha.1 (2019-06-12)`